### PR TITLE
adding documentation for using education endpoint

### PR DESCRIPTION
**Pull request by nirajacharya2**
This PR adds documentation for using education endpoint

related issue: https://github.com/owncloud/ocis-php-sdk/issues/260

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ composer require owncloud/ocis-php-sdk
 Create an Ocis object using the service Url and an access token:
 ```php
 $ocis = new Ocis('https://example.ocis.com', $accessToken);
+// for using eduaction endpoint use education api token
```
Comment (phil-davis) [inline README.md:46]:
```suggestion
// for using education endpoint use education api token
```

File: README.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ composer require owncloud/ocis-php-sdk
 Create an Ocis object using the service Url and an access token:
 ```php
 $ocis = new Ocis('https://example.ocis.com', $accessToken);
+// for using education endpoint use education api token
+$ocis = new Ocis('https://education.ocis.com', null, [], $educationAccessToken);
 ```
```
Comment (individual-it) [inline README.md:48]: I think this is a bit confusing.
On top we are talking about an `access token`
In between there is an `education api token` but the variable is called `$educationAccessToken`
And then afterwards we tell people how to get an access token.

I think you will have to elaborate about
1. that there are two different tokens for different API endpoints
2. that at least one needs to be provided
3. where to get an education access token
4. which token is needed for which functions
5. which token the chapter below is talking about

**Review by individual-it — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -40,18 +40,31 @@ composer require owncloud/ocis-php-sdk
...
 
 ## Getting started
-Create an Ocis object using the service Url and an access token:
+Ocis has two type of access token, One which is used to interact with drive, group, user, shares etc.(OICD access token) and another which can be used to interact with education endpoints (education access token).
```
Comment (individual-it) [inline README.md:43]:
```suggestion
Ocis has two types of access tokens, one which is used to interact with drive, group, user, shares etc.(OICD access token) and another one which can be used to interact with education endpoints (education access token).
```

File: README.md [hunk 4]
```diff
@@ -40,18 +40,31 @@ composer require owncloud/ocis-php-sdk
...
+
+Create an Ocis object using the service Url and an OIDC access token:
 ```php
 $ocis = new Ocis('https://example.ocis.com', $accessToken);
 ```
+Or create an Ocis object to interact with education endpoint using the service Url and an education access token:
```
Comment (individual-it) [inline README.md:49]:
```suggestion
Or create an Ocis object to interact with education endpoints using the service Url and an education access token:
```

File: README.md [hunk 5]
```diff
@@ -40,18 +40,31 @@ composer require owncloud/ocis-php-sdk
...
+```php
+$ocis = new Ocis('https://education.ocis.com', null, [], $educationAccessToken);
+```
 
-Acquiring an access token is out of scope of this SDK, but you can find [examples for that below](#acquiring-an-access-token).
+Atleast one access token should be provided to use the sdk.
```
Comment (individual-it) [inline README.md:54]:
```suggestion
At least one access token should be provided to use the SDK.
```

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**